interface-definitions/include/firewall/tcp-flags.xml.i

```diff
@@ -114,6 +114,23 @@
         </node>
       </children>
     </node>
+    <leafNode name="mss">
+      <properties>
+        <help>Maximum segment size (MSS)</help>
+        <valueHelp>
+          <format>u32:1-16384</format>
+          <description>Maximium segment size</description>
+        </valueHelp>
+        <valueHelp>
+          <format>&lt;min&gt;-&lt;max&gt;</format>
```

Comment:
Maybe we should have two `<valueHelp>` definitions. One for a single number, and another for the range.

Reply:
But there are 2 ValueHelp, aren’t?

Reply:
But there are 2 ValueHelp, aren’t?

Reply:
Sorty - I somehow missed that!